tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,66 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Host Collection CLI"""
+
+from ddt import ddt
+from robottelo.test import CLITestCase
+from robottelo.cli.import_ import Import
+
+
+@ddt
+class TestImport(CLITestCase):
+    """Import CLI tests."""
+
+    """All default tests pass no options to the imprt object
+    In such case methods download a default data set from URL
+    specified in robottelo.properties.
+
+    """
```

Comment:
I'm not sure what happens when two docstrings follow each other. This is a more normal way to write docstrings:

```python
@ddt
class TestImport(CLITestCase):
    """Import CLI tests.

    All default tests pass no options to the imprt object In such case methods
    download a default data set from URL specified in robottelo.properties.

    """
```